### PR TITLE
python3Packages.sentry-sdk: 1.4.3 -> 1.5.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 1.5.0

Change log: https://github.com/getsentry/sentry-python/releases/tag/1.5.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/sentry-sdk/default.nix [hunk 1]
```diff
@@ -55,8 +81,11 @@ buildPythonPackage rec {
   postCheck = "unset NIX_REDIRECTS LD_PRELOAD";
 
   # no tests
-  doCheck = false;
-  pythonImportsCheck = [ "sentry_sdk" ];
+  doCheck = true;
```
Comment (misuzu) [inline pkgs/development/python-modules/sentry-sdk/default.nix:84]: `doCheck = !stdenv.isDarwin;`
Tests are broken on darwin.

Comment (fabaff) [reply pkgs/development/python-modules/sentry-sdk/default.nix]: Thanks for the information.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 146466` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>gns3-server</li>
    <li>home-assistant</li>
    <li>octoprint</li>
  </ul>
</details>
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>alerta-server</li>
    <li>cura</li>
    <li>gns3-gui</li>
    <li>moodle-dl</li>
    <li>moz-phab</li>
    <li>python38Packages.django-rq</li>
    <li>python38Packages.sentry-sdk</li>
    <li>python39Packages.django-rq</li>
    <li>python39Packages.sentry-sdk</li>
    <li>tribler</li>
  </ul>
</details>

**Review by misuzu — APPROVED** [review]
Result of `nixpkgs-review pr 146466` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>octoprint</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>moz-phab</li>
  </ul>
</details>
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>alerta-server</li>
    <li>moodle-dl</li>
    <li>python38Packages.django-rq</li>
    <li>python38Packages.sentry-sdk</li>
    <li>python39Packages.django-rq</li>
    <li>python39Packages.sentry-sdk</li>
  </ul>
</details>

**Comment by collares** [comment]
I am seeing some test failures in `test_rq.py`, such as:

```
=================================== FAILURES ===================================
__________________________________ test_basic __________________________________
tests/integrations/rq/test_rq.py:50: in test_basic
    queue.enqueue(crashing_job, foo=42)
/nix/store/2dx7v83fi90jkw8xkfvwjn6p47mjdc9s-python3.8-rq-1.10.1/lib/python3.8/site-packages/rq/queue.py:508: in enqueue
    return self.enqueue_call(
/nix/store/2dx7v83fi90jkw8xkfvwjn6p47mjdc9s-python3.8-rq-1.10.1/lib/python3.8/site-packages/rq/queue.py:406: in enqueue_call
    return self.enqueue_job(job, pipeline=pipeline, at_front=at_front)
sentry_sdk/integrations/rq.py:107: in sentry_patched_enqueue_job
    return old_enqueue_job(self, job, **kwargs)
/nix/store/2dx7v83fi90jkw8xkfvwjn6p47mjdc9s-python3.8-rq-1.10.1/lib/python3.8/site-packages/rq/queue.py:566: in enqueue_job
    job.save(pipeline=pipe)
/nix/store/2dx7v83fi90jkw8xkfvwjn6p47mjdc9s-python3.8-rq-1.10.1/lib/python3.8/site-packages/rq/job.py:663: in save
    if self.get_redis_server_version() >= (4, 0, 0):
/nix/store/dgmp76d54xxb3bka1fph17yz7n8rfkai-python3-3.8.12/lib/python3.8/distutils/version.py:70: in __ge__
    c = self._cmp(other)
/nix/store/dgmp76d54xxb3bka1fph17yz7n8rfkai-python3-3.8.12/lib/python3.8/distutils/version.py:170: in _cmp
    if self.version != other.version:
E   AttributeError: 'tuple' object has no attribute 'version'
```

**Comment by collares** [comment]
Reported upstream at https://github.com/getsentry/sentry-python/issues/1274

**Comment by gebner** [comment]
Let's disable the rq test for now: https://github.com/NixOS/nixpkgs/pull/150020

**Pull request merged**